Dockerfile

```diff
@@ -0,0 +1,34 @@
+#ARG INSTALL_PYTHON_VERSION=${INSTALL_PYTHON_VERSION:-3.8.12}
```

Comment:
Are these leftover artifacts from testing?

Reply:
yeah, should be gone now